vizro-core/src/vizro/models/_components/table.py

```diff
@@ -108,7 +108,7 @@ def build(self):
             html.Div(
                 [
                     html.H3(self.title, className="table-title") if self.title else None,
-                    html.Div(self.__call__(data_frame=pd.DataFrame()), id=self.id),
+                    html.Div(self.__call__(), id=self.id),
```

Comment:
Please could you add an explanatory comment here that refers to the one in `AgGrid.build` or just copies and pastes it in? (I don't know if the pagination stuff is also relevant here or if it's just done the same way to match AgGrid, but I was confused just reading through the code why we did this here rather than just returning an empty `DataTable`)